herast/tree/patterns/expressions.py

```diff
@@ -336,4 +318,12 @@ def __generate_expression_patterns():
 		if name in vars(module):
 			continue
 		vars(module)[name] = type(name, (AbstractBinaryOpPat,), {'op': op})
+	
+	for op in mem_expressions_ops:
```

Comment:
I think, that generating code for various expressions is not preferred over explicit classes for patterns... It loses some fine control like docstrings documentation, code linting in editors. Later there will also be more features, that are harder to code generate prettily (but is fine for now).

Reply:
The missing code linting made it harder for me to understand before I checked how the code was working, so having more of them didn't seem like a problem to me
If those operands were explicit it would help and I agree there will be no need for generating them